daemon/c_seccomp/adjtimex.c

```diff
@@ -200,7 +194,11 @@ c_seccomp_emulate_settime(c_seccomp_t *seccomp, struct seccomp_notif *req,
 	resp->error = 0;
 	resp->val = ret_settime;
 
+	ret_settime = 0;
```

Comment:
why is this required?

Reply:
the emulation handler needs to exit with zero to signal the kernel that the emulation was successful.
the return value from the syscall itself could also be a positive value (read/write) return number ob written bytes. This is set in the resp->val field. clock_settime does only return 0 on success and -1 on error, thus indeed it is not necessary. But to be safe if someone will copy paste this if he wants to emulate another syscall,...

Reply:
Ok we also do not have this in the other emulation handlers. So for consistency it should be removed ;)